### DISCUSSION TITLE
with unknowed type, failed in  create_aggregate_function_replace_load and type is: Nullable(JSONB)

### DISCUSSION THREAD

**Discussion by Wangxiaobin123** [apache/doris#15174] [A - General / Q&A]
从Kafka导入一条数据，load 异常高，并且BE 节点会宕机，报错信息如下，
with unknowed type, failed in  create_aggregate_function_replace_load and type is: Nullable(JSONB)

**Comment by zy-kkk** [comment]


**Reply by chanson-chan** [reply]
我也遇到了这个问题, 请问有任何进展吗

**Reply by LemonLiTree** [reply]
> 我也遇到了这个问题, 请问有任何进展吗

你现在哪个版本？

**Reply by chanson-chan** [reply]
1.2.0 遇到这个问题, 刚刚升到 1.2.1 试了下没报错了
